server/http/helloapp/command.go

```diff
@@ -0,0 +1,153 @@
+package helloapp
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/mattermost/mattermost-plugin-apps/server/apps"
+	"github.com/mattermost/mattermost-plugin-apps/server/utils/httputils"
+	"github.com/mattermost/mattermost-plugin-apps/server/utils/md"
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/pkg/errors"
+)
+
+type SubCommand struct {
+	AppID       string        `json:"app_id"`
+	Pretext     string        `json:"pretext"`
+	Description string        `json:"description"`
+	SubCommands []SubCommand  `json:"sub_commands"`
+	Args        []interface{} `json:"args"`
+	FormURL     string        `json:"form_url"`
+}
+
+func (h *helloapp) handleHelloCommandSubmission(w http.ResponseWriter, req *http.Request, claims *apps.JWTClaims, call *apps.Call) (int, error) {
+	var e = func(err error, msg string) (int, error) {
+		httputils.WriteJSON(w, apps.CallResponse{
+			Type:     apps.CallResponseTypeError,
+			Markdown: md.MD(errors.Wrap(err, msg).Error()),
+		})
+		return http.StatusInternalServerError, err
+	}
+
+	username := call.Values.Get("user")
+	message := call.Values.Get("message")
+	u, err := h.apps.Mattermost.User.GetByUsername(username)
```

Comment:
We shouldn't use the mattermost api in the helloapp (we should use the way the app would to get this info), but I understand we can leave it like this for now.